plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/WatchManager.java

```diff
@@ -39,13 +39,23 @@
 
 public class WatchManager {
 
+	public static final String[] KINDS = new String[] { 
+			//ResourceKind.BUILD,
```

Comment:
Why BUILD are not watched ?

Reply:
Builds are not editable (no editor that needs to watch them), Properties are not watching any resources, there's no occurrence where I see builds being watched. So why watch them?

Reply:
@jeffmaury looking into it in more details I see you being right: added builds in again.